python/katana/galois.pyx

```diff
@@ -1,7 +1,11 @@
 from .cpp.libgalois.Galois cimport setActiveThreads as c_setActiveThreads
+from .cpp.libgalois.Galois cimport getVersion as c_getVersion
+
 
 _katana_runtime = _katana_runtime_wrapper()
 
 def setActiveThreads(int n):
     return c_setActiveThreads(n)
 
+def getVersion():
+    return c_getVersion()
```

Comment:
This should be called `get_version` to match PEP8. I set a bad example with `setActiveThreads`, that should be changed to `set_active_threads`, though not in this PR.

Reply:
Thank you for the suggestions. I have made the changes with the latest commit.